rebasehelper/__init__.py

```diff
@@ -22,10 +22,21 @@
 #          Nikola Forró <nforro@redhat.com>
 #          František Nečas <fifinecas@seznam.cz>
 
+import logging
 import os
 
 import pkg_resources
 
+try:
+    from rebasehelper.logger import CustomLogger
+except ModuleNotFoundError:
+    # setup.py is being run, dependencies have not been installed
```

Comment:
None of those conditions is necessarily true. Thinking about it now, perhaps simple `# bootstrap` would be sufficient.